job-service-db/src/main/resources/procedures/reportProgress.sql

```diff
@@ -17,9 +17,51 @@
 /*
  *  Name: report_progress
  *
- *  REMOVED
+ *  Description:
+ *  Reports a task progress to the database
  */
 DROP FUNCTION IF EXISTS report_progress(
     in_task_id VARCHAR(58),
     in_status job_status
 );
+CREATE OR REPLACE PROCEDURE report_progress(
+    in_partition_id VARCHAR(40),
+    in_job_id VARCHAR(58),
```

Comment:
Not needed - work it out from the task id.

Reply:
For the job id, I now use internal_get_job_id() to get it from the task but doesn't it add work in the function when we already have the info above.
The task doesn't provide the partition, so we need to pass it I think